sources/packages/web/src/components/students/applicationTracker/MultipleDisbursementBanner.vue

```diff
@@ -29,9 +29,14 @@
     background-color="error-bg"
   >
     <template #content
-      ><span class="font-bold">Reason from your institution:</span>
-      {{ coeDenialReason }}. Please note any scheduled payment(s) will be
-      cancelled.</template
+      ><span class="font-bold"
+        >Your institution declined your enrolment status.</span
+      >
+      Reason from your institution: {{ coeDenialReason }}. All disbursements
```

Comment:
As discussed, this message can be applicable for single disbursement as well.

Reply:
Yes, I am checking with business. Thanks for bringing this up.

Reply:
Update the PR description and send a message to the business for final confirmation on this.

Reply:
Business confirmed that we can use the message for the first disbursement in singular, as below.
![image](https://github.com/bcgov/SIMS/assets/61259237/d176d0e8-3bae-4b41-b8f9-4f41d8967de6)
@CarlyCotton FYI